doc/rest.json

```diff
@@ -279,23 +279,17 @@
     },
     "/taoDeliveryRdf/RestDelivery/update": {
       "post": {
-        "description": "Update a delivery from a valid delivery uri",
+        "description": "Update a delivery by fields",
```

Comment:
please use something more descriptive:
Update parameters of all the deliveries found by criteria given from url query